shared/utility/motion/InverseKinematics.cpp

```diff
@@ -337,23 +337,14 @@ namespace utility::motion::kinematics {
         joints.insert(joints.end(), joints2.begin(), joints2.end());
         return joints;
     }
-    std::vector<std::pair<ServoID, double>> calculateCameraLookJoints(const Eigen::Vector3d& cameraUnitVector) {
-        std::vector<std::pair<ServoID, double>> positions;
-        positions.push_back(std::make_pair(ServoID::HEAD_YAW, std::atan2(cameraUnitVector.y(), cameraUnitVector.x())));
+    std::vector<std::pair<ServoID, float>> calculateHeadJoints(const Eigen::Vector3f& cameraUnitVector) {
+        std::vector<std::pair<ServoID, float>> positions;
```

Comment:
You could always template this function to allow for some flexibility
```suggestion
    template <typename Scalar>
    std::vector<std::pair<ServoID, Scalar>> calculateHeadJoints(const Eigen::Matrix<Scalar, 3, 1>& cameraUnitVector) {
        std::vector<std::pair<ServoID, Scalar>> positions;
```